static/app/views/relocation/getStarted.tsx

```diff
@@ -0,0 +1,146 @@
+import {useContext, useState} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {Button} from 'sentry/components/button';
+import SelectControl from 'sentry/components/forms/controls/selectControl';
+import Input from 'sentry/components/input';
+import {RelocationOnboardingContext} from 'sentry/components/onboarding/relocationOnboardingContext';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import {space} from 'sentry/styles/space';
+import testableTransition from 'sentry/utils/testableTransition';
+import StepHeading from 'sentry/views/relocation/components/stepHeading';
+
+import {StepProps} from './types';
+
+function GetStarted(props: StepProps) {
+  const regions = ConfigStore.get('regions');
+  const [region, setRegion] = useState(regions[0].name);
+  const [orgSlugs, setOrgSlugs] = useState('');
+  const relocationOnboardingContext = useContext(RelocationOnboardingContext);
+
+  const handleContinue = (event: any) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    relocationOnboardingContext.setData({formData});
+    props.onComplete();
+  };
+  return (
+    <Wrapper>
+      <StepHeading step={1}>{t('Basic Information Needed to Get Started')}</StepHeading>
+      <motion.div
+        transition={testableTransition()}
+        variants={{
+          initial: {y: 30, opacity: 0},
+          animate: {y: 0, opacity: 1},
+          exit: {opacity: 0},
+        }}
+      >
+        <Form onSubmit={handleContinue}>
+          <p>
+            {t(
+              'In order to best facilitate the process some basic information will be required to ensure sucess with the relocation process of you self-hosted instance'
+            )}
+          </p>
+          <RequiredLabel>{t('Organization Slugs')}</RequiredLabel>
+          <Input
+            type="text"
+            name="org-slugs"
+            aria-label="org-slugs"
+            onChange={evt => setOrgSlugs(evt.target.value)}
+            required
+            minLength={1}
```

Comment:
What is the actual min-length of a single org? We should set that here too.

Reply:
it's 3 characters, updated